libraries/AR_Motors/AP_MotorsUGV.cpp

```diff
@@ -729,12 +736,43 @@ void AP_MotorsUGV::output_skid_steering(bool armed, float steering, float thrott
     float steering_scaled = steering / 4500.0f; // steering scaled -1 to +1
     float throttle_scaled = throttle * 0.01f;  // throttle scaled -1 to +1
 
+    // sanitize values for asymmetry of thrust, mixer assumes forward thrust is always larger than reverse
+    const float thrust_asymmetry = MAX(_thrust_asymmetry, 1.0);
+    const float min_throttle = -1.0 / thrust_asymmetry;
+
+    // Maximum steering is half way between upper and lower limits
+    const float best_steering_throttle = (1.0 + min_throttle) * 0.5;
```

Comment:
We also have _throttle_max.. not sure if that should be used in place of 1.0.

Reply:
Currently the throttle output is constrained by `_throttle_max` before the mix, so we will never output a higher average throttle. But because were still using 1 here we can output motor than that the left or right motors due the to additional throttle needed for steering. 

Currently were using `_throttle_max` as the maximum combined throttle output.

By changing that 1.0 (and some others) we could change it to be the maximum output to either motor.